code-for-canada/project-thundercat
# [Background Tab] Image Replacement

# Description

Replaced screenshots of images with actual images.
Now a visible difference between EN and FR images

## Type of change

- Code cleanliness or refactor
## Screenshot

![image](https://user-images.githubusercontent.com/2746350/53906022-e44a2280-4017-11e9-9d29-da15060d5454.png)

![image](https://user-images.githubusercontent.com/2746350/53906033-ed3af400-4017-11e9-9174-d3c7077b5e52.png)

![image](https://user-images.githubusercontent.com/2746350/53906053-fb891000-4017-11e9-8cf0-8818697a5b9b.png)

![image](https://user-images.githubusercontent.com/2746350/53906072-0479e180-4018-11e9-9d1b-b9d97a299f26.png)


# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Go to Prototype -> eMIB Test
2.  Pass pre-test
3.  Verify images are present when in background Tab and change when language is toggled.

Screenshot of unit tests run passing:
![image](https://user-images.githubusercontent.com/2746350/53906158-3428e980-4018-11e9-804c-07007cedb19e.png)

# Checklist

Applicable for all code changes.
